x-pack/plugins/security_solution/public/detections/components/rules/description_step/helpers.tsx

```diff
@@ -536,26 +537,25 @@ export const buildAlertSuppressionDescription = (
       )}
     </EuiFlexGroup>
   );
-  if (license.isAtLeast(minimumLicenseForSuppression)) {
-    return [
-      {
-        title: label,
-        description,
-      },
-    ];
-  } else {
-    return [
-      {
-        title: (
-          <>
-            {label}&nbsp;
-            <EuiToolTip position="top" content={i18n.ALERT_SUPPRESSION_INSUFFICIENT_LICENSE}>
-              <EuiIcon type={'alert'} size="l" color="#BD271E" />
-            </EuiToolTip>
-          </>
-        ),
-        description,
-      },
-    ];
-  }
+
+  const title = (
+    <>
+      {label}
+      <EuiBetaBadge
+        label={i18n.ALERT_SUPPRESSION_TECHNICAL_PREVIEW}
+        style={{ verticalAlign: 'middle', marginLeft: '8px' }}
+      />
```

Comment:
Design nit: Beta badge seems to be a bit of a chonker, consider adding `size="s"`. I don't see any [specific design guidelines](https://elastic.github.io/eui/#/display/badge#beta-badge-type) on badges & titles, so please ignore if as-designed 🙂 

<img width="350" alt="image" src="https://user-images.githubusercontent.com/2946766/204649591-5b0c5a24-9e5b-4402-9e24-17c3132eb615.png"> vs <img width="350" alt="image" src="https://user-images.githubusercontent.com/2946766/204649713-a49db540-d3f6-433c-b876-1b036190dfc6.png">

```suggestion
      <EuiBetaBadge
        label={i18n.ALERT_SUPPRESSION_TECHNICAL_PREVIEW}
        style={{ verticalAlign: 'middle', marginLeft: '8px' }}
        size="s"
      />
```

Reply:
Also, I noticed this other `Technical Preview` badge had a tooltip with more information -- not sure if this is necessary here, but just wanted to note. (This badge looks quite beefy as well too, but is at least smaller than the corresponding title 😅 )

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/2946766/204650624-e1d5c3dc-ceaa-4e28-b52a-60d913c18334.png">

Reply:
Good point, I like the smaller one as well - it should do better on small window sizes as well.